portfolio/src/main/webapp/quiz.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Question data type.
+ */
+function Question(question, options, correct) {
```

Comment:
Reading the comment makes it feel like `Question` might be better modeled as a class. WDYT? I think that would also help self document the fields...

Reply:
You are right, classes would be a better choice.